globals/strings.gd

```diff
@@ -239,6 +239,8 @@ func build_choose_cards_string(from_zone, to_zone, amount_min, amount_max,
 			to_zone_str = tr("YOUR_ARCHIVE")
 		"backstage":
 			to_zone_str = tr("YOUR_BACKSTAGE")
+		"bottom_of_deck":
+			to_zone_str = "%s of %s" % [tr("the bottom"), tr("YOUR_DECK")]
```

Comment:
The %s of %s really needs to be translated for this to make sense in all languages.
We should probably just tr the whole phrase and not do a string format here.